go/obscurocommon/l1_types.go

```diff
@@ -108,9 +108,13 @@
 
 type EncryptedSharedEnclaveSecret []byte
 
+type EncodedAttestationReport []byte
```

Comment:
Given there's no methods, do we need to have a type for []byte ?

Reply:
Yeah wasn't sure about that. I was copying the EncodedRollup and the EncryptedSharedEnclaveSecret. I feel like you get type safety with it this way, and it means if we represented it some other way in future, we wouldn't have to change it in the places it's being passed around etc. But happy to be talked out of it tbh, wasn't 100% sure what was the idiomatic way to do it.